plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -126,7 +126,8 @@ def add_pod_disruption_budget
         metadata: {
           name: kubernetes_role.resource_name,
           namespace: resource.dig(:metadata, :namespace),
-          labels: resource.dig_fetch(:metadata, :labels).dup
+          labels: resource.dig_fetch(:metadata, :labels).dup,
+          annotations: (resource.dig(:metadata, :annotations) || {}).dup
```

Comment:
Could move the `dup` inside parens to `resource.dig(:metadata, :annotations).dup`

Reply:
could do `.dig()&.dup || {}` ... but I"m not making another build for that :D